samples/bluetooth/hci_usb/usbd_next_prj.conf

```diff
@@ -15,3 +15,143 @@ CONFIG_USBD_BT_HCI=y
 CONFIG_LOG=y
 CONFIG_USBD_LOG_LEVEL_WRN=y
 CONFIG_UDC_DRIVER_LOG_LEVEL_WRN=y
+
+CONFIG_BT_BUF_EVT_RX_COUNT=16
+
+CONFIG_BT_BUF_EVT_RX_SIZE=255
+CONFIG_BT_BUF_ACL_RX_SIZE=255
+CONFIG_BT_BUF_ACL_TX_SIZE=251
+CONFIG_BT_BUF_CMD_TX_SIZE=255
+
+# Host and Controller common dependencies
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_PER_ADV=y
+CONFIG_BT_PER_ADV_SYNC=y
+CONFIG_BT_PER_ADV_SYNC_MAX=2
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+
+# Broadcast and Connected ISO
+CONFIG_BT_ISO_BROADCASTER=y
+CONFIG_BT_ISO_SYNC_RECEIVER=y
+CONFIG_BT_ISO_CENTRAL=y
+CONFIG_BT_ISO_PERIPHERAL=y
+
+# ISO Streams
+CONFIG_BT_ISO_TX_MTU=310
+CONFIG_BT_ISO_RX_MTU=310
+CONFIG_BT_ISO_MAX_CHAN=4
+CONFIG_BT_ISO_TX_BUF_COUNT=1
+CONFIG_BT_ISO_RX_BUF_COUNT=1
+
+# Controller
+CONFIG_BT_LL_SW_SPLIT=y
```

Comment:
Again, no specific Controller implementation to be referenced in main prj.conf file.